### ISSUE TITLE
Feature Request: Files Rename

### ISSUE THREAD

**Issue by rmpf** [pdfarranger/pdfarranger#678]
Hi. Thanks for all your efforts on this excellent software tool.

I'm wonder is it possible to include an option to rename the selected file.

For example:

I just scanned multiple documents and as a result I obtain a pdf like this 04062022_Scans Color.
Pdfarranger helped me to split the result in individuals files. Next step would be rename those files. We are talking of a lot of files (up to 800). So the alternative to open each one to see its title for latter rename is really aggressive. A better alternative would be to remain in your tool panel that allows me to see a preview and give the new name from there and organize all. Example Rename 04062022_Scans Color PDF1 to car, 04062022_Scans Color PDF2 to car2, 04062022_Scans Color PDF3 to car3, ......to animal, animal2, animal3, app, app2, app3. Afterwards, from there, rejoin those with the same name as one.

Thanks in advance.

**Comment by dreua** [comment]
What happens if you rename the file before opening it in PDF Arranger?

**Comment by dreua** [comment]
I don't think we are going to implement this in our spare time for free. Sounds like a special business use case for me.

Alternate: One can rename files relatively fast with a good file manager, for example with GNOME's Nautilus this would be hitting Space for preview, Space to close it, F2, type name, Enter, Down to the next file. You'll still need to type 800 names, maybe some automation with OCR might be a good idea if you are doing this on a regular basis.

- Let's give this 3 weeks. If nothing productive comes up it should be closed and tagged `stale feature`.

**Comment by dreua** [comment]
I just remembered that our "Export selection to a single file" format does ask for a filename, why are you not using that?
The language barrier makes it hard for me to understand your issue, I didn't get it at all when reading for the first time. Maybe try explaining step by step what commands you are using in PDF Arranger.

**Comment by rmpf** [comment]
Thanks for the prompt response.  It's not really a business issue, it's more a personal hobbie. To be more specific, the documents I scanned were my music scores that I no longer wanted on paper but digitally.  As I mentioned i scanned all the scores and the copier sends one file with all of them that's why I used pdfarranger to separate (split) them into several files which have to be renamed according to the song title.  I know that I can do it from a file manager such as one commander, which, like pdfarranger, allows the preview of the PDF to be able to rename it without having to open it.  Once renamed I have to find each file that belongs to the same song to join them in pdfarranger.  My request was to skip the file manager step since pdfarranger allows the previewing.  I will use the export - save as option once I am going to join all the scores that belong to the same song but I cannot do that until I find all the scores that belong to the same musical work.  I understand how complicated is your time. I appreciate that you were able to evaluate my application.  I congratulate you again for such a good program.  Go ahead.  GB
